From 60515d8fc46eeddc26365f94d053b303aedac13a Mon Sep 17 00:00:00 2001
From: "haotian.qu" <haotian.qu@amlogic.com>
Date: Thu, 1 Jun 2023 17:34:20 +0800
Subject: [PATCH] date: date don't support parameter of '-s' [1/1]

PD#SWPL-124089

Problem:
Run date -s '2022-06-06 15:00:00' fails.

Solution:
Add coreutils feature.

Verify:
local

Signed-off-by: haotian.qu <haotian.qu@amlogic.com>
Change-Id: I4f6c365e5ea7a7be8d7642c5474ae0a74c0d20fc
---
 conf/machine/mesong12b-w400-weston-5.15.config | 1 +
 conf/machine/mesonsm1-ac200-weston-5.15.config | 1 +
 conf/machine/mesont7-an400-weston.config       | 1 +
 conf/machine/mesont7c-an400-weston.config      | 1 +
 4 files changed, 4 insertions(+)

diff --git a/conf/machine/mesong12b-w400-weston-5.15.config b/conf/machine/mesong12b-w400-weston-5.15.config
index faaafe36..27f3acb9 100644
--- a/conf/machine/mesong12b-w400-weston-5.15.config
+++ b/conf/machine/mesong12b-w400-weston-5.15.config
@@ -19,6 +19,7 @@ MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
         wififw-ap6398 \
         ap6xxx \
         npu \
+        coreutils \
         "
 
 RDEPENDS_packagegroup-amlogic-baserootfs_remove = " aml-subtitleserver"
diff --git a/conf/machine/mesonsm1-ac200-weston-5.15.config b/conf/machine/mesonsm1-ac200-weston-5.15.config
index 7c4ea5d0..01ac5cb8 100644
--- a/conf/machine/mesonsm1-ac200-weston-5.15.config
+++ b/conf/machine/mesonsm1-ac200-weston-5.15.config
@@ -19,6 +19,7 @@ MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
         wififw-ap6398 \
         ap6xxx \
         npu \
+        coreutils \
         "
 
 RDEPENDS_packagegroup-amlogic-baserootfs_remove = " aml-subtitleserver"
diff --git a/conf/machine/mesont7-an400-weston.config b/conf/machine/mesont7-an400-weston.config
index c1616088..50b674a5 100644
--- a/conf/machine/mesont7-an400-weston.config
+++ b/conf/machine/mesont7-an400-weston.config
@@ -20,6 +20,7 @@ MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
         ap6xxx \
         optee-linux-driver \
         adla \
+        coreutils \
         "
 
 PREFERRED_VERSION_ap6xxx = "5.4"
diff --git a/conf/machine/mesont7c-an400-weston.config b/conf/machine/mesont7c-an400-weston.config
index 4c19105b..dbdce0d9 100644
--- a/conf/machine/mesont7c-an400-weston.config
+++ b/conf/machine/mesont7c-an400-weston.config
@@ -20,6 +20,7 @@ MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
         ap6xxx \
         optee-linux-driver \
         adla \
+        coreutils \
         "
 
 PREFERRED_VERSION_ap6xxx = "5.4"
-- 
2.25.1

